### PR TITLE
suite-nonsplit-upselling-products

### PR DESCRIPTION
**Pull request by himanshu-pal-metacube**
https://spryker.atlassian.net/browse/CC-16847  
suite-nonsplit-upselling-products

### CODE REVIEW

File: tests/api/suite_nonsplit/glue/general_product_endpoints/upselling_products/negative.robot [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    When I send a GET request:    /carts//up-selling-products
+    Then Response status code should be:    403
+    And Response reason should be:    Forbidden
+    And Response should return error code:    002
+    And Response should return error message:    Missing access token.
+
+Get_upselling_products_using_cart_of_other_customer
+    [Setup]    Run Keywords    I get access token for the customer:    ${yves_user.email}
+    ...  AND    I set Headers:    Authorization=${token}
+    ...  AND    Save value to a variable:    [data][attributes][accessToken]    first_user_token
```
Comment (Katerynaa-Lavryk) [inline tests/api/suite_nonsplit/glue/general_product_endpoints/upselling_products/negative.robot:49]: I suggest saving cartId to reuse in further steps as accessToken is not used anywhere

Comment (himanshu-pal-metacube) [reply tests/api/suite_nonsplit/glue/general_product_endpoints/upselling_products/negative.robot]: Removed

File: tests/api/suite_nonsplit/glue/general_product_endpoints/upselling_products/positive.robot [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+    ...    AND    Cleanup all items in the cart:    ${cart_id}
+    When I send a GET request:    /carts/${cart_id}/up-selling-products
+    Then Response status code should be:    200
+    And Response reason should be:    OK
+    And Response should contain the array of a certain size:    [data]    0
+    And Response body has correct self link
+    [Teardown]    Run Keyword    Cleanup all items in the cart:    ${cart_id}
+
+
+Guest user cart
```
Comment (Katerynaa-Lavryk) [inline tests/api/suite_nonsplit/glue/general_product_endpoints/upselling_products/positive.robot:166]: I guess this line should be commented

Comment (himanshu-pal-metacube) [reply tests/api/suite_nonsplit/glue/general_product_endpoints/upselling_products/positive.robot]: Commented the line

**Review by Katerynaa-Lavryk — APPROVED** [review]

**Pull request merged**